src/libstd/sys/redox/ext/net.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![stable(feature = "unix_socket", since = "1.10.0")]
+#![stable(feature = "unix_socket_redox", since = "1.27.0")]
```

Comment:
Stuff is always initially unstable when added to the standard library.

Reply:
Should I make a tracking issue for it and everything?